validation/src/main/scala/hmda/validation/rules/lar/validity/V570.scala

```diff
@@ -0,0 +1,21 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ Failure, Result }
+import hmda.validation.rules.EditCheck
+
+object V570 extends EditCheck[LoanApplicationRegister] {
+  override def name: String = "V570"
+
+  override def apply(lar: LoanApplicationRegister): Result = {
+    when(lar.lienStatus is equalTo(1)) {
+      lar.rateSpread is equalTo("NA") or {
+        try {
```

Comment:
OK, I did a thing. It doesn't feel entirely beautiful, but it is in one place, and I think even the right place.